### PR TITLE
e

### PR DESCRIPTION
**Pull request by bluebear94**

### CODE REVIEW

File: catalogues/repohost.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+GitHub
+BitBucket
+GitLab
+Codeberg
+Sourcehut
```
Comment (ramidzkh) [inline catalogues/repohost.txt:5]: maybe we can add `{any person}'s gitlab` or similar?

**Review by ramidzkh — APPROVED** [review]
I'll merge in a few days if you don't reply to the comment

File: catalogues/repohost.txt (hunk 1 above)
Comment (bluebear94) [reply catalogues/repohost.txt:5]: What about `{silent unique mod m}{unique author[m]} makes the {unique repohost} repo for {m} private` in `drama`?

**Comment by ramidzkh** [comment]
" a few days" :skull: he forgor

**Pull request merged**